lib/flipper/middleware/memoizer.rb

```diff
@@ -60,7 +60,11 @@ def memoize?(request)
       def memoized_call(env)
         reset_on_body_close = false
         flipper = env.fetch(@env_key) { Flipper }
-        original = flipper.memoizing?
+
+        # Already memoizing. Maybe the memization middleware is mounted twice?
+        # This instance does not need to do anything.
+        return @app.call(env) if flipper.memoizing?
```

Comment:
Should this warn or do something to log that it's skipping memoization?

Reply:
I like the idea of warning.